<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyDialog</class>
 <widget class="QDialog" name="KeyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="generateKeyButton">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>195</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate New Key</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="emailLE">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>155</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>example@site.com</string>
   </property>
  </widget>
  <widget class="QLabel" name="orLabel">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>115</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>OR</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="continueButton">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>75</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Use existing keys</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
